src/main/java/com/enonic/app/guillotine/graphql/GraphQLApi.java

```diff
@@ -187,13 +189,13 @@ private void generateGuillotineApi( GraphQLTypesRegister typesRegister )
         typesRegister.addCreationCallback( "Query", guillotineQueryCreationCallback );
 
         typesRegister.addResolver( "Query", "guillotine", environment -> {
-            final Map<String, Object> localContext = environment.getLocalContext();
+            final Map<String, Object> localContext = new HashMap<>();
```

Comment:
Maybe better to use `Map.of` and don't add `null` vales?
Generally it will be safer to live without `null`.
One side effect - `null` will have to remove the parent mappings when merging in `createDataFetcherResult`

Reply:
Done. We do not allow `null` values.